lib/pager.js

```diff
@@ -0,0 +1,33 @@
+/** Class for paging Bundles. */
+class Pager {
+  /**
+   * Create a pager for the provided client.
+   *
+   * @param {HttpClient} httpClient a configured instance of http client.
+   */
+  constructor(httpClient) {
+    this.httpClient = httpClient;
+  }
+
+  /**
+   * Return the next page of results.
+   *
+   * @param {object} results a bundle result of a FHIR search
+   */
+  nextPage(results) {
+    const { url } = results.link.find(link => link.relation === 'next');
+    return this.httpClient.get(url);
+  }
+
+  /**
+   * Return the previous page of results.
+   *
+   * @param {object} results a bundle result of a FHIR search
+   */
+  prevPage(results) {
+    const { url } = results.link.find(link => link.relation === 'previous');
+    return this.httpClient.get(url);
+  }
+}
+
+module.exports = { Pager };
```

Comment:
Given the name of the file, and the fact that this is the "main" / only thing this file does, perhaps:

`module.exports = Pager`

(analogous to the "default" export from standard ES2015+ ?)

Reply:
Can we have a discussion about this? I would like to know the best practices on `module.export`